datahub-web-react/src/graphql/glossaryTerm.graphql

```diff
@@ -4,6 +4,28 @@ query getGlossaryTerm($urn: String!) {
         type
         name
         hierarchicalName
+        isRelatedTerms {
+            entities {
+                entity {
+                    urn
+                    type    
+                }
+            }
+        }
+        hasRelatedTerms {
+            entities {
+                entity {
+                    urn
+                    type    
+                }
+            }
+        }
+        downstreamLineage {
+            ...downstreamRelationshipFields
+        }
+        upstreamLineage {
+            ...upstreamRelationshipFields
+        }
```

Comment:
We probably don't want to add downstreamLineage and upstreamLineage to the glossary term

Reply:
yup, will remove it from here